Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:11-jdk-slim-bullseye
+FROM openjdk:21-jdk-slim-bullseye
```

Comment:
maybe use alternative distro other than openjdk?

Reply:
What's the issue with OpenJDK, and do you have a recommendation? I don't usually do a lot of Java development to be honest so I'm not very familiar with the alternatives :sweat_smile:

Reply:
it's not about the jdk version but more of an issue with dockerfile and docker image

i'd personally use https://hub.docker.com/_/amazoncorretto

Reply:
if you look at https://hub.docker.com/_/openjdk it shows it's deprecated @adamreeve

Reply:
:+1: makes sense thanks, I wasn't aware it was deprecated. I've changed the base image to be Amazon Corretto now.